### PR TITLE
Add CMT instructions

### PR DESCRIPTION
**Pull request by eloots**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,65 @@ We start from an existing Scala 2 application: an [[Akka](https://akka.io)] acto
...
+- Clone the [Course Management Tools (CMT)](https://github.com/lightbend/course-management-tools)
+- Start an `sbt` session in the cloned CMT repo
+- Run the _studentify_ command and provide two arguments:
+    - The first one is the _absolute_ path to the root folder of the master repository of the exercises
+    - The second one is the absolute path of the folder in which the studentified version will be created
+- Once the _studentify_ command is finished, you can _cd_ into the studentified versions root folder and start an _sbt_ session. You will now be able to use all the student commands listed in the previous section.
+
+### How to _"linearize"_/_"delinearize"_ a master repository
+
+In the context of the _"Moving from Scala 2 to Scala 3"_ course, _linearize_ allows one to view the changes between the different exercises. In other words, this feature allows one to see exactly what needs to be done to _"solve"_ the exercise — so; if you want to actually learn something, don't use this feature — instead, give the exercises a try first...
```
Comment (ckipp01) [inline README.md:52]:
```suggestion
In the context of the _"Moving from Scala 2 to Scala 3"_ course, _linearize_ allows one to view the changes between the different exercises. In other words, this feature allows one to see exactly what needs to be done to _"solve"_ the exercise. So, if you want to actually learn something, don't use this feature — instead, give the exercises a try first...
```

File: README.md [hunk 2]
```diff
@@ -16,3 +16,65 @@ We start from an existing Scala 2 application: an [[Akka](https://akka.io)] acto
...
+- Once the _linearize_ command is finished, _cd_ into the _linearized_ version's root folder and start refactoring your code using interactive rebasing (e.g. `git rebase -i --root`).
+- To apply all changes made in the _linearized_ repo to the master repository, execute the `delinearize` command supplying two arguments:
+  - The first one is the _absolute_ path to the root folder of the master repository of the exercises
+  - The second one is the absolute path of the `linearized` project folder
+- Repeat the _interactive rebasing_ / _delinearization_ sequence as many times as you want
+
+> NOTES
+> 
+> - During the process of interactive rebasing, make sure NOT to change the commit message (which in fact is the exercise's project name) or to squash commits as this will make it impossible to _delinearize_ the project.
+> - Editing code directly in the _master repo_ and editing code via _linearize_/_delinearize_ are mutually exclusive processes. In practice, this means that when you've been using `linearize`/`delinearize` and you decided to make a change directly to files in the exercises in the master repository, you should discard the `linearized` artifact.
```
Comment (ckipp01) [inline README.md:77]:
```suggestion
> - Editing code directly in the _master repo_ and editing code via _linearize_/_delinearize_ are mutually exclusive processes. In practice, this means that when you've been using `linearize`/`delinearize` and you decide to make a change directly to files in the exercises in the master repository, you should discard the `linearized` artifact.
```

File: README.md [hunk 3]
```diff
@@ -16,3 +16,65 @@ We start from an existing Scala 2 application: an [[Akka](https://akka.io)] acto
...
+    - `restoreState <exerciseID>`: restore a previously save state
+- A _linearized_ repository: this is a git repository in which every exercise in the master repository is turned into a commit:
+  - Used to edit the master repository: In general this is a scratch pad repository that is discarded once it has served its purpose. In this use case, interactive rebasing on the linearized repo is used to transform the code across a range of exercises. Once this rebasing is finished, all applied changes can be applied on the master repo by a process of `delinearization`
+  - **There is a second use case for a linearized repo: it can can be used to inspect the differences between consecutive exercises (using a tools such as `SourceTree` or `GitKraken`)**
+
+_Studentification_, _linearization_, and _delinearization_ as well as some administrative tasks (such as renumbering exercises), is done using the so-called [Course Management Tools](https://github.com/lightbend/course-management-tools).
+
+### How to _"studentify"_ a master repository
+
+1. Choose a folder in which the _studentified_ version will be
```
[the remark this answers is not in the document]
Comment (eloots) [reply README.md:42]: @ckipp01 This should probably read:  _~Choose a~ Create a folder_

[the remark this answers is not in the document]
Comment (ckipp01) [reply README.md:42]: Ahhh, yes, sorry. Meant to replace the choose with _Create_, not add it. 👍

[the remark this answers is not in the document]
Comment (eloots) [reply README.md]: Changed it in [06c95df](https://github.com/lunatech-labs/lunatech-scala-2-to-scala3-course/pull/16/commits/06c95dfe44e7d491aaffbb7f653e0019e321117f)

**Pull request merged**